app/views/pull_requests/pull_request_size_prs_repository/index.erb

```diff
@@ -0,0 +1,18 @@
+<%= render 'development_metrics/development_metrics_sidebar', enabled_users_section: @enabled_users_section %>
+  <div class="metrics-container">
+    <div class="row title_sidebar">
+        <div class="col-md-6">
+          <b class="rute_title_grey"><%= params[:repository_name] +'/'%></b>
```

Comment:
Why is the param variable in the view? It's not better to filter it on the controller? 🤔

Reply:
Fix with repository name validation [commit e8ae885d811055f77f917b04d981b2ae730e7d72](https://github.com/rootstrap/rs-code-review-metrics/pull/527/commits/e8ae885d811055f77f917b04d981b2ae730e7d72)